discovery-provider/src/tasks/users.py

```diff
@@ -240,6 +242,10 @@ def parse_user_event(
         user_record.is_creator = event_args._isCreator
     elif event_type == user_event_types_lookup["update_is_verified"]:
         user_record.is_verified = event_args._isVerified
```

Comment:
Can `event_args._isVerified` be false here? If so we should conditionally dispatch the event

Reply:
It defaults to false and we set it to true when a user connects twitter/instagram w/ a verified account. 
I don't think we have never set it to false, but in theory we can - good point, will add an extra check to ensure to only enqueue if true